examples/protocol_specification_ex/fipa.yaml

```diff
@@ -0,0 +1,25 @@
+name: fipa
+author: fetchai
+version: 0.1.0
+license: Apache 2.0
+description: 'A protocol for FIPA ACL.'
+speech_acts:
+  cfp:
+    query: Query
+  propose:
+    proposal: list
+    price: float
+  accept_w_inform:
+    info: dict
+  match_accept_w_inform:
+    info: dict
+  inform:
+    info: dict
+  accept: {}
+  decline: {}
+  match_accept: {}
+
+  # Differences:
+  # propose -> proposal is a list of Descriptions. Declare each d in proposal is of type description.
+  # dialogue_reference is missing
```

Comment:
Please add it, it is required.

Reply:
Note, this points at a limitation in the current spec, if I want to add a field in each performative I need to repeat myself...

Reply:
Done